File: dataset/output/CLAUDE_claude-3-5-sonnet-20240620/Task141_CLAUDE_claude-3-5-sonnet-20240620.java

import java.util.Scanner;

public class ReverseString {
    public static String reverseString(String s) {
        return new StringBuilder(s).reverse().toString();
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a string: ");
        String input = scanner.nextLine();
        System.out.println("Reversed string: " + reverseString(input));
        scanner.close();
    }
}

File: dataset/output/CLAUDE_claude-3-5-sonnet-20240620/Task141_CLAUDE_claude-3-5-sonnet-20240620.c

#include <stdio.h>
#include <string.h>

void reverseString(char* str) {
    int length = strlen(str);
    int i, j;
    char temp;
    for (i = 0, j = length - 1; i < j; i++, j--) {
        temp = str[i];
        str[i] = str[j];
        str[j] = temp;
    }
}

int main() {
    char input[100];
    printf("Enter a string: ");
    fgets(input, sizeof(input), stdin);
    input[strcspn(input, "\\n")] = 0;  // Remove newline character
    reverseString(input);
    printf("Reversed string: %s\\n", input);
    return 0;
}
